napari_btrack/track.py

```diff
@@ -0,0 +1,47 @@
+import btrack
+import napari
+
+from btrack.utils import segmentation_to_objects
+from magicgui import magicgui
+from magicgui.widgets import FunctionGui
+from pathlib import Path
+from typing import Optional
+
+def run_tracker(objects, config_file_path):
```

Comment:
Should this be decorated with `@thread_worker` to avoid blocking `napari` whilst the tracking is running?

Reply:
It's not that straight forward, I'll keep it as it is (the function would not return the same) and create a separate issue for its implementation. (#17)